### PR TITLE
R1/beta2!

### PR DESCRIPTION
**Pull request by waddlesplash**
TODO:
 - [x] Checksums, torrents, final links!
 - [x] Screenshot (and better text?) for WebPositive section
 - [x] Screenshot (and better text?) for HaikuPorts section.

### CODE REVIEW

**Comment by waddlesplash** [comment]
CC @humdingerb, @nielx, @diversys. I'd like to release later today, so, feedback appreciated. If one of you has a lot of ported software installed or a good way to show off Web+, please feel free to take care of those TODOs :)

**Comment by pulkomandy** [comment]
TODO:
- Screenshot of Installer with the Haiku logo
- Screenshot of mouse preferences was done with corrupt mouse settings (shows "B" and "P" instead of 4 and 5 on extra mouse buttons)
- Screenshot of activitymonitor/aboutsystem says "8 core" in AboutSystem but shows only 2 in ActivityMonitor, is that intentional?

**Comment by diversys** [comment]
Some screenshots are still not cropped/have bad quality.

**Comment by pulkomandy** [comment]
Some are also rather useless. I don't think the "meta key" checkbox in preferences is very useful, for example. On the other hand a screenshot of FirstBootPrompt using Thai text could be nice?

File: content/get-haiku/r1beta2/_index.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+</thead>
+<tbody>
+    <tr>
+      <td class="location"><b>Location:</b> East Coast, United States <br/>Provided by: <a target="_blank" class="ext" href="https://haiku-inc.org" title="Haiku, Inc.">Haiku, Inc.</a></td>
+      <td><a class="track" href="https://s3.wasabisys.com/haiku-release/r1beta2/haiku-r1beta2-x86_gcc2_hybrid-anyboot.zip">zip</a></td>
+      <td><a class="track" href="https://s3.wasabisys.com/haiku-release/r1beta2/haiku-r1beta2-x86_64-anyboot.zip">zip</a></td>
+    </tr>
+    <tr>
+      <td class="location"><b>Location:</b> Amsterdam, Netherlands <br/>Provided by: <a target="_blank" class="ext" href="https://haiku-inc.org" title="Haiku, Inc.">Haiku, Inc.</a></td>
+      <td><a class="track" href="https://cdn.haiku-os.org/haiku-release/r1beta2/haiku-r1beta2-x86_gcc2_hybrid-anyboot.zip">zip</a></td>
```
Comment (diversys) [inline content/get-haiku/r1beta2/_index.md:73]: Amsterdam links seem broken.

File: content/get-haiku/r1beta2/release-notes.md [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+
+Additionally, there is now support for mice with more than three buttons, including support for button remapping.
+
+### WebPositive improvements
+
+Haiku's built-in web browser, WebPositive, now uses a newer version of WebKit. Additionally, it contains a number of fixes to the Haiku backend, most notably to reserve significantly less memory, and to fix WebSockets, as well as a variety of crashes.
+
+### More ported software
+
+Thanks to Haiku's relatively robust POSIX compliance, as well as ports of open source GUI toolkits, even more pieces of popular open source software have been ported to Haiku since Beta 1 and are available in the package repositories.
```
Comment (threedeyes) [inline content/get-haiku/r1beta2/release-notes.md:79]: Screenshots:
1. Ported applications:
PortedApps

2. Games
Games

3. Games (opensource and free only)
FreeGamesOnly

**Comment by humdingerb** [comment]
Installer with beta2 logo: https://oshi.at/dTtrsh/installer.png
Mouse prefs with correctly numbered buttons: https://oshi.at/jUoeJP/mouse.png

File: content/get-haiku/r1beta2/_index.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+pkgman full-sync
+```
+
+After that step has completed succesfully, you should immediately reboot the system. You can do this from the Deskbar, or by running ```shutdown -r``` in Terminal.
+
+### Download a Haiku Image
+
+The provided images are in our `anyboot` format. They can be written directly to a <a href="/guides/installing/making_haiku_usb_stick">USB flash drive</a>, an empty disk, or <a href="/get-haiku/burn-cd">written to DVD</a> media.<br />
+The images can be used "live" or used to install Haiku to another disk/partition of your choice.<br />
+They can also be used directly from QEMU or installed in other <a href="/guides/virtualizing">virtual machine</a>.
```
Comment (humdingerb) [inline content/get-haiku/r1beta2/_index.md:49]:
```suggestion
They can also be used directly from QEMU or installed in other <a href="/guides/virtualizing">virtual machines</a>.
```

File: content/get-haiku/r1beta2/_index.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+<p>To make sure that you have got the right file and that it has not been corrupted during file transfer, it is always wise to ensure the integrity of your download. To that end, you can use the SHA256 checksums shown below to verify that what you have is the correct file.</p>
+
+<pre>
+FIXME haiku-r1beta2-x86_64-anyboot.zip
+FIXME haiku-r1beta2-x86_gcc2_hybrid-anyboot.zip
+</pre>
+
+<p>
+	If you are in macOS or Linux, you can use the <code>sha256sum</code>
```
Comment (humdingerb) [inline content/get-haiku/r1beta2/_index.md:99]:
```suggestion
	If you are in Haiku, macOS or Linux, you can use the <code>sha256sum</code>
```

File: content/get-haiku/r1beta2/_index.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+	command from the Terminal to generate a checksum for your downloaded
+	file and verify against those above. If you are in Windows, you can use
+	a tool such as hashtab to display checksums for files directly in explorer.
```
Comment (humdingerb) [inline content/get-haiku/r1beta2/_index.md:102]:
```suggestion
	a tool such as <a href="https://implbits.com/products/hashtab/" title="Download hashtab">hashtab</a> to display checksums for files directly in explorer.
```

File: content/get-haiku/r1beta2/_index.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+	<a href="http://implbits.com/products/hashtab/" title="Download hashtab">download from here</a>.
```
Comment (humdingerb) [inline content/get-haiku/r1beta2/_index.md:103]: remove.

File: content/get-haiku/r1beta2/release-notes.md [hunk 7]
```diff
@@ -0,0 +1,236 @@
...
+Below are two screenshots, showing ActivityMonitor and AboutSystem with a 12pt font size on the left (Haiku's default), and with an 18pt font size on the right (in other words, 150% of "normal.")
+
+<style>td>img{max-width:400px;}</style>
+<table><tr><td><img src="/files/get-haiku/r1beta2/normal-activitymonitor-aboutsystem.png"></td><td><img src="/files/get-haiku/r1beta2/hidpi-activitymonitor-aboutsystem.jpg"></td></tr></table>
+
+There are still some bugs to be ironed out (such as scrollbars not drawing with proper scaling applied, as seen in this screenshot), but things are now generally usable under HiDPI. Other applications, such as Deskbar and Tracker, saw significant improvements to their behavior on HiDPI, as well.
+
+### Deskbar improvements
+
+The Deskbar has had various improvements. It now has a mini-mode, where the icons on the shelf expand leftwards. You can activate it by dragging the shelf space by the right handle to the top of the screen.
```
Comment (humdingerb) [inline content/get-haiku/r1beta2/release-notes.md:54]: Depending on where at "the top of the screen", you may end up with the Deskbar stretched to the top of the screen, Mac style.

```suggestion
The Deskbar has had various improvements. It now has a mini-mode, where the icons on the shelf expand leftwards. You can activate it by dragging the shelf space by the right handle onto the blue leaf.
```

File: content/get-haiku/r1beta2/release-notes.md [hunk 8]
```diff
@@ -0,0 +1,236 @@
...
+
+<img src="/files/get-haiku/r1beta2/deskbar_minimode.png"/><br>
+<small><i>New mini-mode to put the Deskbar at the top of the screen.</i></small>
+
+Furthermore, there are more options for showing larger icons, which is especially useful on screens with a higher DPI.
```
Comment (humdingerb) [inline content/get-haiku/r1beta2/release-notes.md:59]: Was the slider already in Beta1? Instead, maybe point to the improved "Auto-raise" that now also "auto-lowers" when the mouse leaves the Deskbar area. The screenshot below would become unnecessary.

File: content/get-haiku/r1beta2/release-notes.md [hunk 9]
```diff
@@ -0,0 +1,236 @@
...
+
+Haiku's built-in web browser, WebPositive, now uses a newer version of WebKit. Additionally, it contains a number of fixes to the Haiku backend, most notably to reserve significantly less memory, and to fix WebSockets, as well as a variety of crashes.
+
+### More ported software
+
+Thanks to Haiku's relatively robust POSIX compliance, as well as ports of open source GUI toolkits, even more pieces of popular open source software have been ported to Haiku since Beta 1 and are available in the package repositories.
+
+### Installation improvements
+
+Installer now supports excluding optional packages that are included on the installation medium from an actual installation.
```
Comment (humdingerb) [inline content/get-haiku/r1beta2/release-notes.md:83]:
```suggestion
Installer now supports excluding optional packages that are included on the installation medium.
```

File: content/get-haiku/r1beta2/_index.md (hunk 1 above)
Comment (waddlesplash) [reply content/get-haiku/r1beta2/_index.md:73]: When I uploaded this, both links were broken. CC @nielx, did you figure out how to upload to here?

File: content/get-haiku/r1beta2/release-notes.md (hunk 2 above)
Comment (waddlesplash) [reply content/get-haiku/r1beta2/release-notes.md:79]: These are great, thanks! I'll use the first and last of these.

Comment (threedeyes) [reply content/get-haiku/r1beta2/release-notes.md:79]: Something on the last too many penguins ;) Maybe make a new one with free & opensource games where there will be fewer penguins?

Comment (pulkomandy) [reply content/get-haiku/r1beta2/release-notes.md:79]: I think non-free games are fine too? Why not?

Comment (threedeyes) [reply content/get-haiku/r1beta2/release-notes.md:79]: New screnshot without tuxes )
screenshot72

**Pull request merged**